pysd/py_backend/vensim/vensim2py.py

```diff
@@ -230,6 +239,9 @@ def get_equation_components(equation_str):
         - *component* - normal model expression or constant
         - *lookup* - a lookup table
         - *subdef* - a subscript definition
+        - *data* -  a data variable
+
+    keyword: basestring or None
```

Comment:
Could you explain more about what `keyword` is/does, and why it is returned?

Reply:
for DATA type variables, you can pass an argument in vensim in this way

```
observable:HOLD BACKWARD::=
    GET DIRECT DATA(...) ~~|
```
the keyword in this case is the "HOLD BACKWARD" argument, which says hoy the interpolation should be done for DATA objects.

Reply:
That makes sense. Thanks. =)